cmd/cloud/installation_operation.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+//
+
+package main
+
+import "github.com/spf13/cobra"
+
+func init() {
+	installationOperationCmd.AddCommand(installationRestorationOperationCmd)
+	installationOperationCmd.AddCommand(installationDBMigrationOperationCmd)
+}
+
+var installationOperationCmd = &cobra.Command{
+	Use:   "operation",
```

Comment:
`operation` is a pretty vague/general term for this family of commands. Please correct me if this is not the correct understanding of usage, e.g. :

`cloud installation operation list`

I don't think that this command makes it clear what kind of operation the user is listing. Maybe `migration` is a better sub-command for this family?

`cloud installation migration list`

or `cloud installation db-migration` ?

Reply:
The examples are in PR description. The `operation` is just subcommand grouping other commands that reference operations, so it is like that:
```
cloud installation operation db-migration ...
cloud installation operation restoration ...
```

Reply:
So yes, I did understand the usage. "Operation" is very generic and I don't understand its value in the command structure. The resulting command doesn't make much sense to me; everything under `installation` is already an operation. These are db operations; shouldn't the command they are grouped under communicate that?

Reply:
Thanks for the clarification Ian.

So I went with the `operation` subcommand as from the Provisioner side those operations are different than regular create, list, delete, etc. There are actually structs called `DBMigrationOperation` and `DBRestorationOperation` and data for them is preserved in DB. Those are returned as part of commands, they also work in a similar manner therefore I wanted to group them under a single concept of `operation`.

In the example of `cloud installation operation db-migration list` I find it easier to reason about it this way as I am listing `operations` not `db migrations`. 

I think that in the use-case of Provisioner where users of CLI are basically developers/SREs it is better to not abstract that as much as one would in the case of a tool designed to hide the complexity from the user and keep it closer to the internal mechanism as I think it is easier then to understand the system from the perspective of developer (you do not need to traverse different levels of abstractions). 

That said if that would be a more client-facing application I agree with your suggestion.

Anyway, I am 1/5 on that. I am going to move forward with the PR, as it's there for a while but I am open to discussing it further and potentially changing.